Gemfile.lock

```diff
@@ -162,6 +163,11 @@ GEM
       rack (>= 1.1)
       rubocop (>= 0.90.0, < 2.0)
     ruby-progressbar (1.11.0)
+    sanger_warren (0.3.0.pre.rc1)
```

Comment:
If a gem version number contains letters (in this case rc1, for release-candidate 1) then rubygems flags it as a pre-release. This lets me still install it via ruby-gems for uat etc. rather than changing to specifying a git commit.

I'll update it to a full release after code-review UAT